src/views/Space/Delegates.vue

```diff
@@ -118,15 +118,15 @@ watchEffect(() => {
             </tr>
           </thead>
           <td v-if="loading" colspan="3">
-            <UiLoading class="p-4 block text-center" />
+            <UiLoading class="p-4 block" />
```

Comment:
Vertical padding should be size 3 to be consistent with others pages.
```suggestion
            <UiLoading class="px-4 py-3 block" />
```